### PR TITLE
Lab7and8 Peer Review

### PR DESCRIPTION
**Pull request by jenniv27**
@williamdolsen

### CODE REVIEW

**Comment by jenniv27** [comment]
Notes to self.
Lab 7: 
- Specify the transverse momentum (pt) range of the dataset I am using
- Fix approximation part (remove question mark and also specify variables N_higgs and N_qcd as the initial expected yields
- Consider changing density to False in plots, and labeling y axis as event counts instead of probability density?
- Recap final process of splits on mass and d2

Lab 8:
- Explain that the high lumi and low lumi files have pt in the same range as my simulation dataset so I will apply the event selection I determined in Lab 7
- fix N_exp_sliced, ratios, to account for expected yields (I did not previously)

**Comment by williamdolsen** [comment]
Screen Shot 2021-12-13 at 10 30 08 AM
In lab 7 for this formula I would denote the right side of these equations with a N_{new,sim,higgs} or N_{new,sim,qcd}.

**Comment by williamdolsen** [comment]
Screen Shot 2021-12-13 at 10 34 34 AM
For this part I would update similarly to what you recommended to me.  Likely say N_{sim,new,higgs}/\sqrt{N_{sim,new,qcd}}

**Comment by williamdolsen** [comment]
Not something that has to change but when you were cutting on d2 keeping data between 0 to 1.3 gave you a sigma of 6.4266.  Is there any reason you choose the cut 0 to 1.4 with a sigma of 5.9679 sigma instead of the other one?  Even if you stick with the split 0 to 1.4 it may be worth commenting on why you choose it as - from a significance standpoint - it's not the best cut.

**Comment by williamdolsen** [comment]
Rest of lab 7 looks really good!  Great job.

**Comment by williamdolsen** [comment]
For your plots in lab 8 I would keep the Higgs histogram bars orange and the QCD blue.  You established a language in lab 7 for which color represents which data and you should hold that representation

**Comment by williamdolsen** [comment]
for your statistical question in lab 8 problem 1: "What is the significance of my observation given the background is Poisson-distributed, where a stronger signal is given by a larger observation of events?" I would reword the end of this statement, I feel a bit disconnected reading it.

Maybe something like "where a stronger signal is achieved with an observation of more events"?

Same comment for lab 8 problem 2

**Comment by williamdolsen** [comment]
I think that your introduction for lab 8 part 3 is outdated.  You also still have a portion in the report for part 4 which you may want to remove.

**Comment by williamdolsen** [comment]
Great job on the rest of the lab Jennifer!

**Comment by jenniv27** [comment]
@AlexSchuy

**Comment by jenniv27** [comment]
@AlexSchuy
